3_visualize/src/visualize_data.R

```diff
@@ -0,0 +1,71 @@
+############## Visualize data  ######################
+
+project_output_dir <- '3_visualize/out'
+
+dir.create(project_output_dir)
+
+
+plot_results <- function() {
+png(file = file.path(project_output_dir, 'figure_1.png'), width = 8, height = 10, res = 200, units = 'in')
```

Comment:
could you add indentation to make this file easier to read? One easy way to do this is with `Ctrl-a` to select all the text, then `Ctrl-i` to use RStudio's automatic indenting feature. What I'm looking for is that all lines within a function definition should be indented by at least 2 spaces, or more if they're continuations of preceding lines.

Reply:
outdated